recipes/onnx/all/conanfile.py

```diff
@@ -113,7 +113,6 @@ def generate(self):
         # https://cmake.org/cmake/help/v3.28/module/FindPythonInterp.html
         # https://github.com/onnx/onnx/blob/1014f41f17ecc778d63e760a994579d96ba471ff/CMakeLists.txt#L119C1-L119C50
         tc.variables["PYTHON_EXECUTABLE"] = sys.executable.replace("\\", "/")
-        tc.variables["ONNX_BUILD_BENCHMARKS"] = False
```

Comment:
Approving because old versions were never setting it to True and it has now been removed